### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by ziwon**
Thank you for opening this great sprint session. It was nice and quite fun.
If you find imprecise words in translations or any errors, please leave a comment. Maybe you can correct it directly.

### CODE REVIEW

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+"the preview network. <br> In future, the mainnet will be able to select "
+"all jobs offered by Dungeon World. </li>\n"
+"    <li> It takes about 15 seconds for the player activation. <br> But "
+"sometimes it takes longer, so please be patient. 🙏 </li>\n"
+"    <li> If you have Gold, you can become more powerful player. <br>\n"
+"        <a href=\"https://nekoyu.me/token.html\" target=\"_blank\">→ Buy "
+"Gold</a></li>\n"
+"</ul>\n"
+msgstr ""
+"<ul>    <li>猫夢はダンジョンワールドルールを使用しています。<br>ダンジョンワールドの基本的なルールを熟知していただければ、 "
```
Comment (longfin) [inline nekoyume/translations/ja/LC_MESSAGES/messages.po]: I think that we should choose Japanese canonical title. (e.g. '猫夢' or 'ネコユメ')

In general, Japanese title in Katakana is more considered as a unique name. so 'ネコユメ' looks better than '猫夢'.

@ziwon @jckdotim Any idea?

Comment (jckdotim) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: @longfin, Got it. Thanks for detailed suggestion. I want to consider this with another japanese contributors too. I think it LGTM right now.

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2018-08-15 15:01+0900\n"
+"PO-Revision-Date: 2018-08-15 15:02+0900\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```
Comment (jckdotim) [inline nekoyume/translations/ja/LC_MESSAGES/messages.po]: @ziwon Could you add your name and mail information here? (If you don't want to add, I think it is better to remove this line.)

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+msgid "Armor"
+msgstr "鎧"
+
+#: nekoyume/templates/dashboard.html:367
+msgid "Food"
+msgstr "食べ物"
+
+#: nekoyume/templates/dashboard.html:373
+msgid "Find monster"
+msgstr "ンスターを探しに出かけます。"
```
Comment (qria) [inline nekoyume/translations/ja/LC_MESSAGES/messages.po]: typo? `ンスター`

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+msgid "Address to send"
+msgstr "送信住所"
+
+#: nekoyume/templates/dashboard.html:425
+msgid "Combine three items to get a new item. (needs 1 GOLD)"
+msgstr "三つのアイテムを組み合わせて、新しいアイテムを取得します。（1 GOLD 必要）"
+
+#: nekoyume/templates/dashboard.html:464
+msgid "Currently in progress ..."
+msgstr "在進行中..."
```
Comment (qria) [inline nekoyume/translations/ja/LC_MESSAGES/messages.po]: possible typo?

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+
+#: nekoyume/templates/dashboard.html:550
+#, python-format
+msgid "You are defeated by %(monster)s..."
+msgstr "%(monster)sに敗北した..."
+
+#: nekoyume/templates/dashboard.html:557
+#, python-format
+msgid "%(monster)s has run away..."
+msgstr "%(monster)sは 逃げてしまった..."
```
Comment (qria) [inline nekoyume/translations/ja/LC_MESSAGES/messages.po]: Spacing intended?

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+msgstr "睡眠を介して体力回復"
+
+#: nekoyume/templates/dashboard.html:648
+#, python-format
+msgid "You delievered %(amount)s of %(item)s."
+msgstr "%(item)s %(amount)sつを送った。"
+
+#: nekoyume/templates/dashboard.html:654
+msgid "Receiver:"
+msgstr "受けた人："
```
Comment (qria) [inline nekoyume/translations/ja/LC_MESSAGES/messages.po]: Maybe present tense is more natural?

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+msgstr "%(item)sを使用したが、効果がなかった。"
+
+#: nekoyume/templates/dashboard.html:600
+#, python-format
+msgid "Level Up! Now, your level is %(lv)s."
+msgstr "レベルアップ！ 今レベル%(lv)sです。"
+
+#: nekoyume/templates/dashboard.html:604
+msgid "Level up was failed"
+msgstr "レベルアップ！失敗"
```
Comment (qria) [inline nekoyume/translations/ja/LC_MESSAGES/messages.po]: `！` is a little confusing here...

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+
+#: nekoyume/templates/dashboard.html:578
+#, python-formats
+msgid "You tried to use %(item)s but it doesn't works."
+msgstr "%(item)sを使用したが、効果がなかった。"
+
+#: nekoyume/templates/dashboard.html:600
+#, python-format
+msgid "Level Up! Now, your level is %(lv)s."
+msgstr "レベルアップ！ 今レベル%(lv)sです。"
```
Comment (qria) [inline nekoyume/translations/ja/LC_MESSAGES/messages.po]: mix of polite and impolite form?

File: nekoyume/translations/ja/LC_MESSAGES/messages.po (hunk 1 above)
Comment (ziwon) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: @longfin It sounds good!

File: nekoyume/translations/ja/LC_MESSAGES/messages.po (hunk 2 above)
Comment (ziwon) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: @jckdotim Ok, then I'm gonna add it right now.

File: nekoyume/translations/ja/LC_MESSAGES/messages.po (hunk 3 above)
Comment (ziwon) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: @qria It's probably a mistake. Fixed it.

File: nekoyume/translations/ja/LC_MESSAGES/messages.po (hunk 6 above)
Comment (ziwon) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: Actually, dunno. But fixed as said.

File: nekoyume/translations/ja/LC_MESSAGES/messages.po (hunk 7 above)
Comment (ziwon) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: Thanks!

File: nekoyume/translations/ja/LC_MESSAGES/messages.po (hunk 8 above)
Comment (ziwon) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: Fixed to "自分の今のレベルは%(lv)s。"

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+"Gold</a></li>\n"
+"</ul>\n"
+msgstr ""
+"<ul>    <li>猫夢はダンジョンワールドルールを使用しています。<br>ダンジョンワールドの基本的なルールを熟知していただければ、 "
+"ゲームにもっと早く慣れることができます。<br>        <a "
+"href=\"http://book.dwgazetteer.com/\" "
+"\"target=\"_blank\">→ ダンジョンワールド英語公開版を見に行く</a></li>    <li>現在プレビューネットでは、職業選択れ"
+"せずに初心者のみの仕事が固定さ<br>今後、メインネットでは、ダンジョンの世界で提供されるすべての職業と種族の選択が可能 "
+"予定です。</li>    <li>プレイヤーの行動が実際に実行されるのには、約15秒の時間がかかります。<br>しかし、時には "
+"さらに時間がかかるもので忍耐を持って待ってください。🙏/li>    <li>ゴールドを保有していれば、より簡単かつ強力な"
```
[the remark this answers is not in the document]
Comment (qria) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: Ah, I was not aware of the korean translation. We should ask the author in gitter.

File: nekoyume/translations/ja/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -0,0 +1,300 @@
...
+"    <li> If you have Gold, you can become more powerful player. <br>\n"
+"        <a href=\"https://nekoyu.me/token.html\" target=\"_blank\">→ Buy "
+"Gold</a></li>\n"
+"</ul>\n"
+msgstr ""
+"<ul>    <li>猫夢はダンジョンワールドルールを使用しています。<br>ダンジョンワールドの基本的なルールを熟知していただければ、 "
+"ゲームにもっと早く慣れることができます。<br>        <a "
+"href=\"http://book.dwgazetteer.com/\" "
+"\"target=\"_blank\">→ ダンジョンワールド英語公開版を見に行く</a></li>    <li>現在プレビューネットでは、職業選択れ"
+"せずに初心者のみの仕事が固定さ<br>今後、メインネットでは、ダンジョンの世界で提供されるすべての職業と種族の選択が可能 "
```
[the remark this answers is not in the document]
Comment (qria) [reply nekoyume/translations/ja/LC_MESSAGES/messages.po]: @ziwon was this comment considered?

**Comment by jckdotim** [comment]
LGTM

**Comment by jckdotim** [comment]
Thank you for your great contribution. :)

**Pull request merged**